src/services/job_handler_interface.py

```diff
@@ -62,6 +62,10 @@ def dmss_sync(self):
         for field, value in merged_kwargs.items():
             setattr(self, field, value)
 
+    def append_log(self, log):
+        new = f"\n{log}" if (self.log and log) else log
+        self.log = self.log + new if self.log else new
```

Comment:
```suggestion
        if log:
          self.log = self.log + f"\n{new}" if self.log else new
```
Isn't this enough? :thinking: